content/concepts/kata/translations.md

```diff
@@ -5,73 +5,124 @@ sidebar: explanation-kata
 
 # Translations
 
-Codewars platform supports a variety of [programming languages](/languages/), and every kata can have more than one language version available. Users can choose any language available to solve the kata, they can also solve one kata in as many languages they wish. After gaining some experience and sufficient privileges, users are allowed to create translations to make kata available for broader group of users.
+Codewars supports a variety of [programming languages](/languages/) and every kata can be available in more than one language. Users may choose any language available to solve the kata and do so in as many languages as they wish. After gaining the required privileges, users who solved the kata are allowed to create or review translations to make the kata available for a larger audience.
 
-## Training on a language version
+## Training on a selected language
 
-When user picks a kata to train on and enters kata trainer view, they can choose the language they want to practice. Some kata are available in many languages, some just in a few, and some only in one or two. If a kata is not available in your favorite language, then unfortunately you have to select another language you know, or wait until someone creates a translation. Completing a kata in only one language is enough to mark the kata as solved, but still, there is something to gain from completing it in other languages too:
+When the user enters the kata trainer to solve a kata, they can select a language to train on. Some kata are available in many languages, some in just a few, and some only in one or two. If a kata is not available in your favorite language, you must select another language you are familiar with or wait until a translation in your preferred language is approved. Completing a kata in only one language is enough to mark the kata as solved; however, there is something to gain from completing it in other languages too:
 
-- your [language rank](/concepts/gamification/ranks/#user-rank-breakdown) progresses for every language you solved the kata with (but Honor and overall rank progress are awarded only once per kata)
-- you unlock the access to solutions of other users in this language
-- you can view full test suite used to test user attepts against their solutions
+- Your [language rank](/concepts/gamification/ranks/#user-rank-breakdown) progresses for every language you solved the kata with (but Honor and overall rank progress are awarded only once per kata).
+- You unlock the access to solutions of other users in this language.
+- You can view full test suite used to test user attempts against their solutions.
 
 ## Creating a translation
```

Comment:
You are right, and I will think how to extract it somewhere. But first I'd like to have this branch merged because I think I did something not right when I created it and now it has a veeery long tail of commits from previous branches. I am afraid that if I start moving files here, I will get conflicts which I will not be able to resolve :)